custom/icds_reports/reports/immunization_coverage_data.py

```diff
@@ -79,8 +79,8 @@ def get_data_for(filters):
             "rightLegend": {
                 "average": (in_month_total * 100) / float(valid_total or 1),
                 "info": _((
-                    "Percentage of children at age 3 who have recieved complete immunization as per "
-                    "National Immunization Schedule of India."
+                    "Percentage of children 1 year+ who have recieved complete immunization as per "
```

Comment:
received is spelled incorrectly. This was preexisting but should be fixed in a future PR